samples/data-grid/basic/cell-format/demo.js

```diff
@@ -0,0 +1,44 @@
+DataGrid.dataGrid('container', {
+    dataTable: {
+        columns: {
+            ID: [1, 2, 3, 4, 5],
+            Name: ['Alice', 'Bob', 'Charlie', 'David', 'Eve'],
+            HireDate: [
+                1579046400000,
+                1532304000000,
+                1552176000000,
+                1622332800000,
+                1510963200000
+            ],
+            Age: [28, 35, 40, 23, 30],
+            Department: ['HR', 'Engineering', 'Sales', 'Marketing', 'Finance'],
+            Salary: [50000, 75000, 55000, 45000, 65000],
+            PerformanceScore: [85, 70, 95, 60, 88]
+        }
+    },
+    columns: [{
+        id: 'HireDate',
+        cells: {
+            formatter: function () {
+                return new Date(this.value).toLocaleDateString();
+            }
```

Comment:
In v12 we should be able to replace this with something like `format: '{value%[YBd]}'`. Maybe add a note somewhere to revisit this when the changes are merged?

Reply:
@karolkolodziej : IMO we should remove this column from the demo until v12 is ready, to avoid showcasing the not recommended `formatter` when not needed.

Reply:
I'll create a ticket in Asana.

Reply:
Just for the records: In Highcharts, we would run `Highcharts.setOptions({ lang })` to set the language strings required for localized dates. In DataGrid, `setOptions` is not exposed, so we don't have control over language. This applies to time formatting, but also number formatting.

Reply:
I will soon add `setOptions` to DataGrid, so it will be possible to solve this problem also this way, if possible I will simply change the lang format source for the core Templating function.